### ISSUE TITLE
Font resizing: Fonts inside of web components don't resize consistently

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#4754]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When using native browser settings to increase font sizes to very large, not all components resize the same.

I also noticed that in the example I give below for the process-list, `h3.usa-process-list__heading` is defined in pixels. If we could change this to rems, the font should resize. 

## What I expected to happen

I expected fonts to resize consistently.

## Reproducing

- CSS Library version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: (e.g. Firefox, IE 11)

**URL(s) where this bug occurs**

- https://design.va.gov/storybook/?path=/docs/uswds-va-process-list--docs

**Steps to reproduce:**

1. Use browser settings to change font size (this is not the same as zooming in)
2. In chrome, go to chrome://settings/?search=font+size
3. Change font to "Very Large"
4. View page in browser
5. Go to https://design.va.gov/storybook/?path=/docs/uswds-va-process-list--docs

**Screenshot:**

<img width="983" height="672" alt="Image" src="https://github.com/user-attachments/assets/09a3ba91-5df2-41e3-b3eb-91963fdc6c6d" />

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [X] This bug is not blocking any work
- [ ] Other

### Details

There is a [slack thread that explains the issue](https://dsva.slack.com/archives/C01DBGX4P45/p1735058549203829), and how teams can fix it. Except in this case, this is inside a web component.

**Comment by amyleadem** [comment]
## Updates
- Sept 15, 2025: [Accessibility specialists Slack thread that captures font resizing findings in the wild on va.gov](https://dsva.slack.com/archives/C0335Q88X0S/p1757958858178239)

**Comment by amyleadem** [comment]
@caw310 The work on this expanded in scope, so I am breaking this task into sub-issues. I am now using this parent issue to capture the effort to create a plan to audit and address font resizing issues in the component library, the forms library, and select pages for VA.gov.
